modules/external-auth-configuring.adoc

```diff
@@ -64,48 +64,55 @@ kind: Authentication
 metadata:
 # ...
 spec:
-  oidcProviders: <1>
+  type: OIDC <1>
+  webhookTokenAuthenticator: null <2>
+  oidcProviders: <3>
   - claimMappings:
+      extra: <4>
+        key: example.com/role
```

Comment:
I missed this before, but this is actually a list of values so this should have:

```suggestion
       - key: example.com/role
```